sdk/include/ndk/rtlfuncs.h

```diff
@@ -4693,6 +4693,54 @@ BOOLEAN
 NTAPI
 RtlGetNtProductType(_Out_ PNT_PRODUCT_TYPE ProductType);
 
+//
+// Synchronization functions
+//
+VOID
```

Comment:
In addition it's most probable these functions need to be also tagged with `NTSYSAPI` if they are exported from ntdll.